src/scaffoldfitter/fitterstepalign.py

```diff
@@ -59,6 +59,7 @@ def __init__(self):
         super(FitterStepAlign, self).__init__()
         self._alignGroups = False
         self._alignMarkers = False
+        self._auto_update_projections = False
```

Comment:
This was needlessly added?

Reply:
No. Not directly part of this PR but it seems reasonably to add it in because it is a minor improvement fix that really doesn't seem worth having its own PR.

Reply:
It isn't used, and it isn't clear what it would be for.